src/etl/sars-cov-2/index.ts

```diff
@@ -0,0 +1,71 @@
+import Airtable, { FieldSet } from "airtable";
+import { pipe } from "fp-ts/lib/function.js";
+import {
+  asyncEtlStep,
+  etlStep,
+  getEnvironmentVariableOrThrow,
+  getMongoClient,
+  writeDataToMongoEtlStep,
+} from "../helpers.js";
+import { validateFieldSetFromAirtableStep } from "./steps/validate-field-set-from-airtable-step.js";
+import { cleanFieldNamesAndRemoveUnusedFieldsStep } from "./steps/clean-field-names-and-remove-unused-fields-step.js";
+import { parseDatesStep } from "./steps/parse-dates-step.js";
+import { latLngGenerationStep } from "./steps/lat-lng-generation-step.js";
+import { removeRecordsThatAreFlaggedNotToSave } from "./steps/remove-records-that-are-flagged-to-not-save-step.js";
+import { jitterPinLatLngStep } from "./steps/jitter-pin-lat-lng-step.js";
+import { transformIntoFormatForDatabaseStep } from "./steps/transform-into-format-for-database-step.js";
+import { filterStudiesThatDoNotMeetDataStructureRequirement } from "./steps/filter-studies-that-do-not-meet-data-structure-requirements.js";
+import { transformNotReportedValuesToUndefinedStep } from "./steps/transform-not-reported-values-to-undefined-step.js";
+
+const runEtlMain = async () => {
+  console.log("Running SarsCoV-2 ETL");
+  const airtableApiKey = getEnvironmentVariableOrThrow({
+    key: "AIRTABLE_API_KEY",
+  });
+  const airtableArboBaseId = getEnvironmentVariableOrThrow({
+    key: "AIRTABLE_SARSCOV2_BASE_ID",
+  });
+  const mongoUri = getEnvironmentVariableOrThrow({ key: "MONGODB_URI" });
+  const databaseName = getEnvironmentVariableOrThrow({ key: "DATABASE_NAME" });
+
+  const mongoClient = await getMongoClient({ mongoUri });
+
+  const airtable = new Airtable({ apiKey: airtableApiKey });
+  const base = new Airtable.Base(airtable, airtableArboBaseId);
+  const estimateSheet = base.table("Rapid Review: Estimates");
+
+  const allEstimatesUnformatted: (FieldSet & { id: string })[] =
+    await estimateSheet
+      .select()
+      .all()
+      .then((estimateSheet) =>
+        estimateSheet.map((record) => ({ ...record.fields, id: record.id }))
```

Comment:
why do we need to do id seperately? Does record.fields not return the id? is id not part of FieldSet. Is it because it is called like estimate_id?

Reply:
Yes that's correct, id is not part of the `FieldSet`. If you check out the way the Airtable library works it just does it like this, I don't think it has to do with anything with our table in particular, it's just how the library works.